platform/ios/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Mapbox welcomes participation and contributions from everyone.  Please read [CON
 
 * Added [quadkey](https://msdn.microsoft.com/en-us/library/bb259689.aspx) support and limited WMS support in raster tile URL templates. ([#5628](https://github.com/mapbox/mapbox-gl-native/pull/5628))
 
+* Map debug options 'overdraw' and 'stencil clip' are now disabled (no-ops) in release mode. ([#5555](https://github.com/mapbox/mapbox-gl-native/pull/5555))
```

Comment:
Refer to the actual symbol names. Also, no need to say "in release mode", as that's the only build configuration we support.